packages/lit-dev-content/site/tutorials/content/async-directive/05.md

```diff
@@ -40,15 +40,16 @@ handle, and use that to start the timer only once:
 
 Start the timer in the `update` lifecycle callback.
 
-<aside class="positive">
+{% aside  "positive" %}
 
-The default implementation of `update` simply calls `render`, but since `update`
-is [_not_ called during
-SSR](https://lit.dev/docs/templates/custom-directives/#differences-between-update()-and-render()),
-`update` is the right spot to start/subscribe to asynchronous tasks we don't
-want running on the server.
+Start or subscribe to asynchronous tasks that you don't want running in the
```

Comment:
Same as the last one, I think we should leave the original text as a no-header aside for now.